examples/step-17/CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ ENDIF ()
 #
 # Are all dependencies fulfilled?
 #
-IF(NOT DEAL_II_WITH_PETSC OR DEAL_II_PETSC_WITH_COMPLEX)
+IF(NOT DEAL_II_WITH_PETSC OR DEAL_II_PETSC_WITH_COMPLEX) # keep in one line
```

Comment:
@bangerth Further, this line has to have the exact format
```
IF(...) # keep in one line
```
But I guess this is not a big issue - after all we all such code tends to get copied from other example steps.